ISISComputingGroup/ibex_gui
# Removed setting of pyepics in the client preferences

PyEpics is no longer required, but still expected by the client. This should fix an issue where the client prompts for PYTHONPATH changes on first start up.

To test:
*Make sure you do not have a directory at C:\Instrument\Apps\Python\Lib\site-packages\epics
*In a fresh install of the client (deleting old workspaces) check that no dialogue pops up about the PYTHONPATH when first starting the client
